Release 0.54 (#10906)
radius-project/radius: docs/release-notes/v0.54.0.md

# Announcing Radius v0.54.0

> Today we're happy to announce the release of Radius vX.Y.Z. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.

Reviewer: ```suggestion
Today we're happy to announce the release of Radius v0.54.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
```

We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!

## Intro to Radius

If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.

## Highlights

> <!-- TALK TO THE PM TEAM ABOUT WHAT HIGHLIGHTS TO ADD HERE -->

Reviewer: ```suggestion
### Fixed `rad workspace show` errors when no current workspace exists
The `rad workspace show` command was updated to provide a more user-friendly experience when no workspace is currently set. Instead of returning an error, the command now logs an informational message to guide the user. See the [CLI docs](https://docs.radapp.io/reference/cli/rad_workspace_show/) for more details.
```

## Breaking changes

<!-- ADD ANY BREAKING CHANGES HERE, IF ANY -->

## New contributors

Welcome to our new contributors who have merged their first PR in this release!

- @filipevrevez made their first contribution in <https://github.com/radius-project/radius/pull/10749>

> ## Upgrading to Radius vX.Y.Z

Reviewer: ```suggestion
## Upgrading to Radius v0.54.0
```
